CRM/Core/Resources.php

```diff
@@ -603,7 +603,11 @@ public function addCoreResources($region = 'html-header') {
 
       $tsLocale = CRM_Core_I18n::getLocale();
       // Dynamic localization script
-      $this->addScriptUrl(CRM_Utils_System::url('civicrm/ajax/l10n-js/' . $tsLocale, array('r' => $this->getCacheCode())), $jsWeight++, $region);
+      $args = [
+        'r' => $this->getCacheCode(),
+        'cid' => CRM_Core_Session::getLoggedInContactID(),
+      ];
+      $this->addScriptUrl(CRM_Utils_System::url('civicrm/ajax/l10n-js/' . $tsLocale, $args, FALSE, NULL, FALSE), $jsWeight++, $region);
```

Comment:
@colemanw is there anything that is needed here to trigger the re-loading if the cid doesn't match or does it do that automatically?

Reply:
The changed value of the cid url param breaks browser caching so it reloads automatically.

Reply:
I think the point is the url will be different & hence not treated the same?

Reply:
yep happy with that just wanted to double check the logic good to merge from me 👍